account_check_printing_report_base/report/lang.py

```diff
@@ -12,9 +14,20 @@ class Num2WordESCustom(Num2Word_ES):
 
     # TODO: PR to remove overwrite in num2words code and use CURRENCY_FORMS
     def to_currency(self, val, longval=True, old=False):
-        return Num2Word_Base.to_currency(
-            self, val, currency="EUR", cents=True, separator=" con", adjective=False
-        )
+        if StrictVersion(
```

Comment:
I prefer the `try` approach, as you are still sticking to something incorrect: what would happen on `0.5.11` version?

Reply:
Well, that can also have some caveats, as I ended up doing a small refactor that I tried to explain in the comments.

Prior to version `0.5.10`, besides the typo, the problem was that the class `Num2Word_ES` (the one we inherit here) in turn inherited from the base class `Num2Word_Base` but overwrote the `to_currency` method. That is, instead of just applying plain Python inheritance, we needed to call the base class directly with `Num2Word_Base.to_currency`.

Now the code has been standardize after https://github.com/savoirfairelinux/num2words/pull/167 and inheritance is now in place as normal, so we can call `super` directly. 

The fact that there are these 2 changes in behavior makes it necessary to work differently according to the version, not only based on the exception thrown. I think this is clearer, but I can always avoid this small refactor and use the old implementation in both situations.

> what would happen on `0.5.11` version?

Assuming the code doesn't suffer massive changes upstream, we would use the new implementation, that I mention above. I can always constraint the version to `<=0.5.10`, which is safer...

Reply:
I really don't see the problem in the try approach and this code is far more convoluted than the other, and we shouldn't pin and library version.